grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -75,6 +81,24 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
       alertReceiveChannelStore.getSearchResult() && alertReceiveChannelStore.getSearchResult()[0];
     if (directPagingAlertReceiveChannel) {
       setSelectedTeamDirectPaging(directPagingAlertReceiveChannel);
+      await alertReceiveChannelStore.updateChannelFilters(directPagingAlertReceiveChannel.id);
+      await store.slackChannelStore.updateItems();
+
+      // Set unique available chatops channels
+      const filterIds = alertReceiveChannelStore.channelFilterIds[directPagingAlertReceiveChannel.id];
+      let availableChannels = [];
+      let channelKeys = new Set();
+      filterIds.map((channelFilterId) => {
+        IntegrationHelper.getChatOpsChannels(alertReceiveChannelStore.channelFilters[channelFilterId], store)
+          .filter((channel) => channel)
+          .map((channel) => {
+            if (!channelKeys.has(channel.name + channel.icon)) {
```

Comment:
it's better to extract this to var `const channelKey = channel.name + channel.icon` and use it below